[TF FE] Document full list of TF operations and their support by TF FE (#20640)
openvinotoolkit/openvino: src/frontends/tensorflow/docs/supported_ops.md

Reviewer: Why we add `docs` directory in frontend when we have list of operations in https://github.com/openvinotoolkit/openvino/blob/5dafee4ac16e1cf757a1ba38008bdf5db0ae0e69/docs/articles_en/about_openvino/compatibility_and_support/supported_operations_framework_frontend.md?plain=1#L749

Author: This one is only for developers to understand the gap. The other one for users sort of.
May be in some future we will merge these two docs.

Reviewer: What is the difference? It looks in the other one we have outdated info

Author: The difference is that users docs contain only supported ops. This one is developer documentation that expresses the gap what we need to contribute more. This table gives us idea about next GFIs and what we can contribute.

# TensorFlow Operations Supported by OpenVINO TensorFlow Frontend 

> Here is a table of supported operations by TensorFlow Frontend from [tf.raw_ops](https://www.tensorflow.org/api_docs/python/tf/raw_ops).
> Supported operation means that TensorFlow Frontend is capable to convert it into OpenVINO representation.

Reviewer 2: ```suggestion
Here is a table of operations supported by the TensorFlow Frontend from [tf.raw_ops](https://www.tensorflow.org/api_docs/python/tf/raw_ops).
A "supported operation" is one that TensorFlow Frontend can convert to the OpenVINO representation.
```
